src/js/components/ServicesTable.js

```diff
@@ -219,26 +219,28 @@ var ServicesTable = React.createClass({
   },
 
   getDestroyConfirmDialog: function () {
-    let message = null;
     const {state} = this;
     let service = state.serviceToChange;
+    let serviceName = '';
     let itemText = 'Service';
     if (service instanceof ServiceTree) {
       itemText = 'Group';
     }
 
     if (service) {
-      message = (
-        <div className="container-pod flush-top container-pod-short-bottom">
-          <h2 className="text-danger text-align-center flush-top">Destroy {itemText}</h2>
-          <p>
-            Are you sure you want to destroy <span className="emphasize">{service.getId()}</span>? This action is irreversible.
-          </p>
-          {this.getErrorMessage()}
-        </div>
-      );
+      serviceName = service.getId();
     }
 
+    let message = (
```

Comment:
If we are always going to have this as children, why not just put it as 'children'?
```js
    return (
      <Confirm
        ...>
        <div className="container-pod flush-top container-pod-short-bottom">
          <h2 className="text-danger text-align-center flush-top">Destroy {itemText}</h2>
          <p>
            Are you sure you want to destroy <span className="emphasize">{serviceName}</span>? This action is irreversible.
          </p>
          {this.getErrorMessage()}
        </div>
      </Confirm>
    );
```